src/plugin/svipc/sysvipc.cpp

```diff
@@ -58,12 +58,10 @@ using namespace dmtcp;
  *  8. Non ckptLeader processes unmap all shmat() addresses corresponding to
  *     the shm-object.
  *  9. BARRIER -- CHECKPOINTED
- * 10. At this point, the contents of the memory-segment have been saved.
```

Comment:
Is there a specific reason you deleted this line? (Sorry, I'm not trying to nitpick here; it's just a genuine question.)